storage/storage-interface/src/cached_state_view.rs

```diff
@@ -23,14 +23,10 @@ use std::{
 
 /// `VerifiedStateView` is like a snapshot of the global state comprised of state view at two
```

Comment:
nit: we'll probably want to update all of the out of date comments referencing the old type name, right?

Reply:
Good eye !